pageserver/src/bin/pageserver.rs

```diff
@@ -134,6 +134,7 @@ fn main() -> anyhow::Result<()> {
     info!(?conf.virtual_file_io_engine, "starting with virtual_file IO engine");
     info!(?conf.virtual_file_io_mode, "starting with virtual_file IO mode");
     info!(?conf.wal_receiver_protocol, "starting with WAL receiver protocol");
+    info!(?conf.validate_wal_contiguity, "starting with WAL contiguity validation");
```

Comment:
nit: for another PR, maybe we should just dump the entire effective config.

Reply:
Let's chat about it with the wider team. I proposed this a while back but I recall some push back